src/main/java/uk/gov/ons/ctp/response/collection/exercise/domain/ExerciseSampleUnit.java

```diff
@@ -70,22 +70,6 @@ public SampleUnitParent toSampleUnitParent(
     return parent;
   }
 
-  // To be removed once actionplans are deprecated
```

Comment:
Not sure if I understand it right. But we will need collection exercise id,  sample unit type and sample unit ref. to be propagated to case. Is this right?

Reply:
There are 2 functions called `toSampleUnitParent` that have different signatures.  The one I'm deleting takes an actionPlanId and collectionExerciseId.
The one that's remaining takes an boolean activeEnrolment and a collectionExerciseId.